bundle/Resources/config/services.yml

```diff
@@ -194,7 +194,10 @@ services:
 
     ezrepoforms.form_processor.content:
         class: "%ezrepoforms.form_processor.content.class%"
-        arguments: ["@ezpublish.api.service.content", "@router"]
+        arguments:
+            - "@ezpublish.api.service.content"
+            - "@ezpublish.api.service.location"
+            - "@router"
```

Comment:
Hmm, how about we gradually change double to single quotes (meaning for service being updated only)?
*I know this makes file inconsistent, but it's the only way to get rid of old convention without making "diff mess".*